src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3874,18 +3874,18 @@
      * </pre>
      *
      * <p>parses as:</p>
-     * <pre>
-     * +--LITERAL_NEW (new)
-     *     |
-     *     +--IDENT (ArrayList)
-     *     +--LPAREN (()
-     *     +--ELIST
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--NUM_INT (50)
-     *     +--RPAREN ())
-     * </pre>
+     *       <pre>
+     * LITERAL_NEW -&t; new
```

Comment:
Change all occurring of `&t;` to `&gt;`

Reply:
Resolved this

Reply:
Resolved, please click resolve from your side.